features/copyCourse.feature

```diff
@@ -1,108 +1,80 @@
-            | sample course | sample topic name | test text content | test description   |
-
-
-    @copyCourseWithGeoGebra
-    Scenario Outline: teacher can copy course with certain GeoGebra
-        Given teacher creates course with name '<courseName>'
-        And teacher chooses course with name '<courseName>'
-        When teacher adds a topic with name '<topicName>'
-        And teacher adds content GeoGebraArbeitsblatt with id '<geogebraID>'
-        And teacher clicks Save-changes button
-        And teacher goes to courses page
-        And teacher chooses course with name '<courseName>'
-        And teacher clicks Duplicate-course button
-        And teacher goes to courses page
-        Then teacher should see that cloned course with name '<courseName> - Kopie' is visible on the list
-        And teacher should see that copied course with name '<courseName> - Kopie' contains topic with name '<topicName>'
-        Examples:
-            | courseName        | topicName | text                  | geogebraID |
-            | sample course geo | geo topic | some sample text here | ucxngdjf   |
+Feature: Copy a created course with different data
 
-    #@copyCourseWithMaterial
-    #Scenario Outline: teacher can copy course with certain Material
-    #Given teacher creates course with name '<courseName>'
-    #And teacher chooses course with name '<courseName>'
-    #And teacher adds a topic with name '<topicName>'
-    #When teacher adds content Material
-    #And teacher clicks Save-changes button
-    #And teacher goes to courses page
-    #And teacher chooses course with name '<courseName>'
-    #And teacher clicks Duplicate-course button
-    #And teacher goes to courses page
-    #Then teacher should see that copied course with name '<courseName> - Kopie' is visible on the list
-    #And teacher should see that copied course with name '<courseName> - Kopie' contains topic with name '<topicName>'
-    #Examples:
-    #| courseName             | topicName      | text                  |
-    #| sample course material | material topic | some sample text here |
+	Background:
+		Given User arrives on the Schul-Cloud homepage
 
+	@simpleCopyCourse
+	Scenario Outline: As a user, I want to be able to copy an existing course
+		When <userRole> logs in
+		And <userRole> performs first login actions: data protection acceptance
+		And <userRole> goes to courses page
+		When <userRole> creates course with name '<courseName>'
+		And <userRole> enters course name '<courseName>' into search field
+		Then <userRole> should see that amount of courses with name '<courseName>' is '1'
+		When <userRole> chooses course with name '<courseName>'
+		And <userRole> clicks Duplicate-course button
+		And <userRole> goes to courses page
+		Then <userRole> should see that cloned course with name '<courseName> - Kopie' is visible on the list
+		And <userRole> should see that amount of courses with name '<courseName> - Kopie' is '1'
```

Comment:
shouldn't 1 also be a dynamic parameter?

Reply:
No. Because this test creates and copy only one course with name <courseName> . That is unchanging.